pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -67,7 +66,7 @@ func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
 		name := b.AutoscalingGroupName(ig)
 
 		if featureflag.SpotinstHybrid.Enabled() {
-			if spotinstmodel.HybridInstanceGroup(ig) {
+			if HybridInstanceGroup(ig) {
```

Comment:
We might want to call this `SpotInstHybridInstanceGroup` if it's spotinst specific